module/PowerShellEditorServices/Start-EditorServices.ps1

```diff
@@ -268,33 +271,36 @@ function Set-NamedPipeMode {
 LogSection "Console Encoding"
 Log $OutputEncoding
 
-function Test-NamedPipeName-OrCreate-IfNull {
+function Get-ValidatedNamedPipeName {
     param(
         [string]
         $PipeName
     )
-    if (-not $PipeName) {
+
+    if (!$PipeName) {
```

Comment:
Perhaps it's worth adding a small comment saying that "if no PipeName is passed in, then we create one that's guaranteed to be valid"

Reply:
Done.